adafruit_bitmapsaver.py

```diff
@@ -116,10 +133,10 @@ def save_pixels(file_or_filename, pixel_source=None, palette=None):
     :param palette: the Palette to use for looking up colors in the bitmap
     """
     if not pixel_source:
-        if "DISPLAY" in dir(board):
-            pixel_source = board.DISPLAY
-        else:
+        if "DISPLAY" not in dir(board):
```

Comment:
it was written with `dir()` originally but it's probably better if this used `hasattr` instead. Do you mind making that change while you're working on this?
so instead of:
```
if "DISPLAY" not in dir(board):
```
it could be:
```
if not hasattr(board, 'DISPLAY'):
```